--- editor:/users/zimooo2/.vscode-server/data/User/History/1e202976/XbzY.sh
+++ /users/zimooo2/.vscode-server/data/User/History/1e202976/dATi.sh
@@ -6,9 +6,9 @@
 declare -ar MEASUREMENT_DEFAULT_WORKLOAD_IDS=(
     "bc-kron.sg-long"
     "pr-kron.sg-long"
-    "bc-twitter.sg-long"
-    "pr-twitter.sg-long"
-    "faiss_10M-long"
+    #"bc-twitter.sg-long"
+    #"pr-twitter.sg-long"
+    #"faiss_10M-long"
 
     #"XSBench"
     #"mg.D.x"
