--- editor:/users/zimooo2/.vscode-server/data/User/History/-198ec96b/isNg.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-198ec96b/e8Wn.sh
@@ -5,14 +5,13 @@
 
 declare -ar MEASUREMENT_DEFAULT_WORKLOAD_IDS=(
     "bc-kron.sg"
+    "pr-kron.sg"
+    "bc-twitter.sg"
+    "pr-twitter.sg"
     "faiss_10M"
-    #"mg.D.x"
-    #"DuckDB-TPCH-sf100"
-
-    #"XSBench"
-    #"pr-kron.sg"
-    #"bc-twitter.sg"
-    #"pr-twitter.sg"
+    "mg.D.x"
+    "DuckDB-TPCH-sf100"
+    "XSBench"
     #"lulesh2.0_s400"
 
 
