--- editor:/users/zimooo2/.vscode-server/data/User/History/-1f82d819/su6r.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-1f82d819/Q2rI.sh
@@ -101,12 +101,12 @@
             #fi
 
             # HybridTier
-            run_measurement_setup "${size}"
-            if [[ -x "${SCRIPT_DIR}/measurement_hybridtier.sh" ]]; then
-                sudo -E "${SCRIPT_DIR}/measurement_hybridtier.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" huge "${workload_id}"
-            else
-                echo "WARNING: ./measurement_hybridtier.sh not found or not executable; skipping HybridTier"
-            fi
+            #run_measurement_setup "${size}"
+            #if [[ -x "${SCRIPT_DIR}/measurement_hybridtier.sh" ]]; then
+            #    sudo -E "${SCRIPT_DIR}/measurement_hybridtier.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" huge "${workload_id}"
+            #else
+            #    echo "WARNING: ./measurement_hybridtier.sh not found or not executable; skipping HybridTier"
+            #fi
 
             #run_measurement_setup "${size}"
             #"${SCRIPT_DIR}/measurement_arms.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "${ARMS_LIB_SUFFIX}" "${workload_id}"
