--- editor:/users/zimooo2/.vscode-server/data/User/History/-46e76ef1/h02o.cpp
+++ /users/zimooo2/.vscode-server/data/User/History/-46e76ef1/Mkpc.cpp
@@ -447,7 +447,7 @@
     {
         this->can_promote = false;
     }
-    if (rank > num_dram_pages + (0.5 * (total_scores - num_dram_pages)))
+    if (rank > num_dram_pages + (01.0 * (total_scores - num_dram_pages)))
     {
         this->can_demote = true;
     }
