--- editor:/users/zimooo2/.vscode-server/data/User/History/-1356624/Y5sp.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-1356624/yS74.sh
@@ -79,7 +79,7 @@
 
             # Model
             if [[ -x "${SCRIPT_DIR}/measurement_model.sh" ]]; then
-                "${SCRIPT_DIR}/measurement_model.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "_train" "${workload_id}"
+                "${SCRIPT_DIR}/measurement_model.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "_long" "${workload_id}"
             else
                 echo "WARNING: ./measurement_model.sh not found or not executable; skipping model"
             fi
