--- editor:/users/zimooo2/.vscode-server/data/User/History/-46e76ef1/b1WQ.cpp
+++ /users/zimooo2/.vscode-server/data/User/History/-46e76ef1/szjq.cpp
@@ -439,7 +439,7 @@
         this->can_promote = false;
     }
 #else
-    if (rank < (num_dram_pages * 0.75))
+    if (rank < (num_dram_pages * 0.5))
     {
         this->can_promote = true;
     }
@@ -447,7 +447,7 @@
     {
         this->can_promote = false;
     }
-    if (rank > (num_dram_pages * 1.25))
+    if (rank > total_scores - (0.5 * (total_scores - num_dram_pages)))
     {
         this->can_demote = true;
     }
