--- editor:/users/zimooo2/.vscode-server/data/User/History/481e7509/bGyo.sh
+++ /users/zimooo2/.vscode-server/data/User/History/481e7509/VpzS.sh
@@ -2,7 +2,7 @@
 
 WORKLOAD_ID="pr-kron.sg-long"
 WORKLOAD_OUTPUT="pr-kron.sg-long"
-WORKLOAD_COMMAND="OMP_NUM_THREADS=16 ${BENCH_ROOT}/gapbs/pr -n 100 -f ${BENCH_ROOT}/gapbs/benchmark/graphs/kron.sg"
+WORKLOAD_COMMAND="OMP_NUM_THREADS=16 ${BENCH_ROOT}/gapbs/pr -n 50 -f ${BENCH_ROOT}/gapbs/benchmark/graphs/kron.sg"
 WORKLOAD_EXE_NAME="pr"
 WORKLOAD_MODEL_BASE="pr-kron.sg"
 WORKLOAD_VIRTUAL_STEP_SAMPLES="10000"
