--- editor:/users/zimooo2/.vscode-server/data/User/History/-4d012c11/S0aa.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-4d012c11/yiX2.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-WORKLOAD_ID="DuckDB-TPCH-sf100"
-WORKLOAD_OUTPUT="DuckDB-TPCH-sf100"
+WORKLOAD_ID="DuckDB-TPCH-sf100-original"
+WORKLOAD_OUTPUT="DuckDB-TPCH-sf100-original"
 WORKLOAD_COMMAND="OMP_NUM_THREADS=16 ${BENCH_ROOT}/duckdb/build/release/benchmark/benchmark_runner benchmark/large/tpch-sf100/.*benchmark --threads=16 --nruns=5 --disable-timeout"
 WORKLOAD_EXE_NAME="benchmark_runner"
 WORKLOAD_MODEL_BASE="DuckDB-TPCH-sf100"
