--- editor:/users/zimooo2/.vscode-server/data/User/History/-198ec96b/VmJf.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-198ec96b/uSUr.sh
@@ -8,13 +8,13 @@
     #"pr-kron.sg"
     #"bc-twitter.sg"
     #"pr-twitter.sg"
-    #"XSBench"
+    "XSBench"
     #
     #"faiss_10M"
     #"mg.D.x"
     #"DuckDB-TPCH-sf100"
-    "bc-twitter.sg-long"
-    "pr-twitter.sg-long"
+    #"bc-twitter.sg-long"
+    #"pr-twitter.sg-long"
 
     #"lulesh2.0_s400"
 
