--- editor:/users/zimooo2/.vscode-server/data/User/History/-1356624/U7PZ.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-1356624/07D5.sh
@@ -73,7 +73,7 @@
             echo "---- Workload ${workload_id} ----"
 
 
-             Model
+            # Model
             if [[ -x "${SCRIPT_DIR}/measurement_model.sh" ]]; then
                 "${SCRIPT_DIR}/measurement_model.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "" "${workload_id}"
             else
