--- editor:/users/zimooo2/.vscode-server/data/User/History/1e202976/W0XF.sh
+++ /users/zimooo2/.vscode-server/data/User/History/1e202976/kd6Z.sh
@@ -4,13 +4,13 @@
 MEASUREMENT_WORKLOADS_PATH="${MEASUREMENT_WORKLOADS_DIR}/workloads"
 
 declare -ar MEASUREMENT_DEFAULT_WORKLOAD_IDS=(
-    #"bc-kron.sg-long"
-    #"pr-kron.sg-long"
-    #"bc-twitter.sg-long"
+    "bc-kron.sg-long"
+    "pr-kron.sg-long"
+    "bc-twitter.sg-long"
     "pr-twitter.sg-long"
-    #"faiss_10M-long"
+    "faiss_10M-long"
 
-    #"XSBench"
+    "XSBench"
     #"mg.D.x"
     #"DuckDB-TPCH-sf100"
 
