--- editor:/users/zimooo2/.vscode-server/data/User/History/-2c506191/kvBD.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-2c506191/oFAI.sh
@@ -77,12 +77,12 @@
             "${SCRIPT_DIR}/measurement_arms.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "" "${workload_id}"
 
 
-            # Model
-            #if [[ -x "${SCRIPT_DIR}/measurement_model.sh" ]]; then
-            #    "${SCRIPT_DIR}/measurement_model.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "" "${workload_id}"
-            #else
-            #    echo "WARNING: ./measurement_model.sh not found or not executable; skipping model"
-            #fi
+             Model
+            if [[ -x "${SCRIPT_DIR}/measurement_model.sh" ]]; then
+                "${SCRIPT_DIR}/measurement_model.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "" "${workload_id}"
+            else
+                echo "WARNING: ./measurement_model.sh not found or not executable; skipping model"
+            fi
 
             # NOMAD
             #if [[ -x "${SCRIPT_DIR}/measurement_nomad.sh" ]]; then
