--- editor:/users/zimooo2/.vscode-server/data/User/History/34ed5a81/I8T2.sh
+++ /users/zimooo2/.vscode-server/data/User/History/34ed5a81/KFns.sh
@@ -8,11 +8,11 @@
     "mg.D.x-original"
     "DuckDB-TPCH-sf100-original"
 
-    "bc-kron.sg"
-    "pr-kron.sg"
-    "bc-twitter.sg"
-    "pr-twitter.sg"
-    "XSBench"
+    #"bc-kron.sg"
+    #"pr-kron.sg"
+    #"bc-twitter.sg"
+    #"pr-twitter.sg"
+    #"XSBench"
 
     #"lulesh2.0_s400"
 
