--- editor:/users/zimooo2/.vscode-server/data/User/History/-5db2003a/0Od1.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-5db2003a/4nY3.sh
@@ -25,10 +25,10 @@
         for workload_id in "${WORKLOAD_IDS[@]}"; do
             echo "---- Workload ${workload_id}: ARMS train then model train ----"
 
-            #run_measurement_setup "${size}"
-            "${SCRIPT_DIR}/measurement_arms.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}_train" "_train" "${workload_id}"
+            run_measurement_setup "${size}"
+            #"${SCRIPT_DIR}/measurement_arms.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}_train" "_train" "${workload_id}"
 
-            #"${SCRIPT_DIR}/measurement_model.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}_train" "_train" "${workload_id}"
+            "${SCRIPT_DIR}/measurement_model.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}_train" "_train" "${workload_id}"
 
             run_measurement_teardown
         done
