--- editor:/users/zimooo2/.vscode-server/data/User/History/-5da7210a/zEei.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-5da7210a/woEH.sh
@@ -2,7 +2,7 @@
 
 WORKLOAD_ID="pr-twitter.sg-long"
 WORKLOAD_OUTPUT="pr-twitter.sg-long"
-WORKLOAD_COMMAND="OMP_NUM_THREADS=16 ${BENCH_ROOT}/gapbs/pr -n 100 -f ${BENCH_ROOT}/gapbs/benchmark/graphs/twitter.sg"
+WORKLOAD_COMMAND="OMP_NUM_THREADS=16 ${BENCH_ROOT}/gapbs/pr -n 50 -f ${BENCH_ROOT}/gapbs/benchmark/graphs/twitter.sg"
 WORKLOAD_EXE_NAME="pr"
 WORKLOAD_MODEL_BASE="pr-twitter.sg"
 WORKLOAD_VIRTUAL_STEP_SAMPLES="10000"
