--- editor:/users/zimooo2/.vscode-server/data/User/History/3b775728/9QUo.sh
+++ /users/zimooo2/.vscode-server/data/User/History/3b775728/F15c.sh
@@ -28,8 +28,8 @@
 #hist_lengths=(4 8)
 
 pcts=(99)
-history_summary_modes=(2)
-hist_lengths=(10)
+history_summary_modes=(0)
+hist_lengths=(3)
 #switch_scalers=(1.0 2.0 4.0 8.0 16.0 32.0 64.0 128.0 256.0)
 #switch_scalers=(256.0 128.0 64.0 32.0 16.0 8.0 4.0 2.0 1.0 )
 switch_scalers=(0.1)
