--- editor:/users/zimooo2/.vscode-server/data/User/History/-73c3a42b/IB7i
+++ /users/zimooo2/.vscode-server/data/User/History/-73c3a42b/toH6
@@ -25,7 +25,7 @@
 DEFAULT_PLATFORM ?= C220G5
 
 # Compile-time configuration matrix
-MODEL_SCORE_HISTORY_SUMMARY_VALUES := 2
+MODEL_SCORE_HISTORY_SUMMARY_VALUES := 1
 # 0=min_max, 1=average, 2=adjusted_moving_average
 HISTORY_LENGTH_VALUES := 10
 # SWITCH_SCALER_VALUES := 0.03125 0.0625 0.125 0.25 0.5 1.0 1.5
