--- editor:/users/zimooo2/.vscode-server/data/User/History/1e202976/0LHJ.sh
+++ /users/zimooo2/.vscode-server/data/User/History/1e202976/XbzY.sh
@@ -4,11 +4,11 @@
 MEASUREMENT_WORKLOADS_PATH="${MEASUREMENT_WORKLOADS_DIR}/workloads"
 
 declare -ar MEASUREMENT_DEFAULT_WORKLOAD_IDS=(
-    "bc-kron.sg"
-    "pr-kron.sg"
-    "bc-twitter.sg"
-    "pr-twitter.sg"
-    "faiss_10M"
+    "bc-kron.sg-long"
+    "pr-kron.sg-long"
+    "bc-twitter.sg-long"
+    "pr-twitter.sg-long"
+    "faiss_10M-long"
 
     #"XSBench"
     #"mg.D.x"
