--- editor:/users/zimooo2/.vscode-server/data/User/History/-44a11d91/XqDX.cpp
+++ /users/zimooo2/.vscode-server/data/User/History/-44a11d91/jAK2.cpp
@@ -1724,7 +1724,7 @@
 #if USE_MODEL == (true)
         auto max_hugepages = max_dram_hugepages(scores);
         // float benefit = SWITCH_SCALER * hot_page->score * HF_SAMPLE_PERIOD * latency_diff;
-        double benefit = (hot_page->score - 40) * HF_SAMPLE_PERIOD * latency_diff;
+        double benefit = (hot_page->score - 80) * HF_SAMPLE_PERIOD * latency_diff;
         // std::cout << "cost: " << cost << " PROMOTION_COST_MULTIPLIER: " << get_promotion_cost_multiplier()
         //           << " MODEL_DISCOUNT_PERCENT: " << MODEL_DISCOUNT_PERCENT
         //           << " promotion_cost_avg: " << promotion_cost_avg << " hot_page->score: " << hot_page->score;
@@ -1766,7 +1766,7 @@
 
 #if USE_MODEL == (true)
                     auto max_hugepages = max_dram_hugepages(scores);
-                    benefit -= (cold_page->score + 40) * HF_SAMPLE_PERIOD * latency_diff;
+                    benefit -= (cold_page->score + 80) * HF_SAMPLE_PERIOD * latency_diff;
 #else
                     benefit -= cold_page->score * hot_page->hot_age * HF_SAMPLE_PERIOD * latency_diff;
 #endif
