--- git:8d6a8bc66:workloads/pr-kron.sg-long.sh
+++ /users/zimooo2/.vscode-server/data/User/History/481e7509/UcUb.sh
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+WORKLOAD_ID="pr-kron.sg-long"
+WORKLOAD_OUTPUT="pr-kron.sg-long"
+WORKLOAD_COMMAND="OMP_NUM_THREADS=16 ${BENCH_ROOT}/gapbs/pr -n 100 -f ${BENCH_ROOT}/gapbs/benchmark/graphs/kron.sg"
+WORKLOAD_EXE_NAME="pr"
+WORKLOAD_MODEL_BASE="pr-kron.sg"
+WORKLOAD_VIRTUAL_STEP_SAMPLES="100000"
+WORKLOAD_SYSTEMS="arms hybridtier model logging"