--- editor:/users/zimooo2/.vscode-server/data/User/History/-44a11d91/iPbT.cpp
+++ /users/zimooo2/.vscode-server/data/User/History/-44a11d91/0Cpk.cpp
@@ -1728,7 +1728,7 @@
             cost += 1 * scores[max_hugepages - 1].page->score * HF_SAMPLE_PERIOD * latency_diff;
         }
         // float benefit = SWITCH_SCALER * hot_page->score * HF_SAMPLE_PERIOD * latency_diff;
-        double benefit = (hot_page->score - 40) * HF_SAMPLE_PERIOD * latency_diff;
+        double benefit = (hot_page->score) * HF_SAMPLE_PERIOD * latency_diff;
         // std::cout << "cost: " << cost << " PROMOTION_COST_MULTIPLIER: " << get_promotion_cost_multiplier()
         //           << " MODEL_DISCOUNT_PERCENT: " << MODEL_DISCOUNT_PERCENT
         //           << " promotion_cost_avg: " << promotion_cost_avg << " hot_page->score: " << hot_page->score;
