--- git:8d6a8bc66:workloads/pr-twitter.sg-short.sh
+++ /users/zimooo2/.vscode-server/data/User/History/100e46f8/BVBR.sh
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+WORKLOAD_ID="pr-twitter.sg-long"
+WORKLOAD_OUTPUT="pr-twitter.sg-long"
+WORKLOAD_COMMAND="OMP_NUM_THREADS=16 ${BENCH_ROOT}/gapbs/pr -n 1 -f ${BENCH_ROOT}/gapbs/benchmark/graphs/twitter.sg"
+WORKLOAD_EXE_NAME="pr"
+WORKLOAD_MODEL_BASE="pr-twitter.sg"
+WORKLOAD_VIRTUAL_STEP_SAMPLES="10000"
+WORKLOAD_SYSTEMS="arms hybridtier model logging"