--- editor:/users/zimooo2/.vscode-server/data/User/History/-198ec96b/xIV4.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-198ec96b/MRqf.sh
@@ -8,11 +8,11 @@
     "pr-kron.sg"
     "bc-twitter.sg"
     "pr-twitter.sg"
-    "XSBench"
+    #"XSBench"
     #
-    "faiss_10M-original"
-    "mg.D.x-original"
-    "DuckDB-TPCH-sf100-original"
+    #"faiss_10M-original"
+    #"mg.D.x-original"
+    #"DuckDB-TPCH-sf100-original"
     #"bc-twitter.sg-long"
     #"pr-twitter.sg-long"
 
