--- editor:/users/zimooo2/.vscode-server/data/User/History/-198ec96b/WXPu.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-198ec96b/mxOc.sh
@@ -13,10 +13,10 @@
     "pr-kron.sg"
     "DuckDB-TPCH-sf100-original"
 
-    "XSBench"
+    #"XSBench"
     ##
-    "faiss_10M-original"
-    "mg.D.x-original"
+    #"faiss_10M-original"
+    #"mg.D.x-original"
     #"bc-twitter.sg-long"
     #"pr-twitter.sg-long"
 
