--- editor:/users/zimooo2/.vscode-server/data/User/History/-198ec96b/8bOi.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-198ec96b/oB7b.sh
@@ -4,18 +4,18 @@
 MEASUREMENT_WORKLOADS_PATH="${MEASUREMENT_WORKLOADS_DIR}/workloads"
 
 declare -ar MEASUREMENT_DEFAULT_WORKLOAD_IDS=(
-    "pr-graph-80GB.sg"
-    "bc-graph-80GB.sg"
+    #"pr-graph-80GB.sg"
+    #"bc-graph-80GB.sg"
 
-    #"DuckDB-TPCH-sf100-original"
-    #"bc-kron.sg"
-    #"pr-kron.sg"
-    #"bc-twitter.sg"
-    #"pr-twitter.sg"
-    #"XSBench"
+    "DuckDB-TPCH-sf100-original"
+    "bc-kron.sg"
+    "pr-kron.sg"
+    "bc-twitter.sg"
+    "pr-twitter.sg"
+    "XSBench"
     ##
-    #"faiss_10M-original"
-    #"mg.D.x-original"
+    "faiss_10M-original"
+    "mg.D.x-original"
     #"bc-twitter.sg-long"
     #"pr-twitter.sg-long"
 
