--- editor:/users/zimooo2/.vscode-server/data/User/History/-2c506191/IlWz.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-2c506191/YagE.sh
@@ -94,11 +94,11 @@
             #fi
 
             # Model
-            #if [[ -x "${SCRIPT_DIR}/measurement_model.sh" ]]; then
-            #    "${SCRIPT_DIR}/measurement_model.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "" "${workload_id}"
-            #else
-            #    echo "WARNING: ./measurement_model.sh not found or not executable; skipping model"
-            #fi
+            if [[ -x "${SCRIPT_DIR}/measurement_model.sh" ]]; then
+                "${SCRIPT_DIR}/measurement_model.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "" "${workload_id}"
+            else
+                echo "WARNING: ./measurement_model.sh not found or not executable; skipping model"
+            fi
 
             # HybridTier
             #run_measurement_setup "${size}"
@@ -112,20 +112,20 @@
             #"${SCRIPT_DIR}/measurement_arms.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "${ARMS_LIB_SUFFIX}" "${workload_id}"
 
             # DRAM-only baseline
-            if [[ -x "${SCRIPT_DIR}/measurement_dram_only.sh" ]]; then
-                run_measurement_setup_baseline_default "${size}"
-                "${SCRIPT_DIR}/measurement_dram_only.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "${workload_id}"
-            else
-                echo "WARNING: ./measurement_dram_only.sh not found or not executable; skipping DRAM-only"
-            fi
+            #if [[ -x "${SCRIPT_DIR}/measurement_dram_only.sh" ]]; then
+            #    run_measurement_setup_baseline_default "${size}"
+            #    "${SCRIPT_DIR}/measurement_dram_only.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "${workload_id}"
+            #else
+            #    echo "WARNING: ./measurement_dram_only.sh not found or not executable; skipping DRAM-only"
+            #fi
 
             # CXL-only baseline
-            if [[ -x "${SCRIPT_DIR}/measurement_cxl_only.sh" ]]; then
-                run_measurement_setup_baseline_default "${size}"
-                "${SCRIPT_DIR}/measurement_cxl_only.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "${workload_id}"
-            else
-                echo "WARNING: ./measurement_cxl_only.sh not found or not executable; skipping CXL-only"
-            fi
+            #if [[ -x "${SCRIPT_DIR}/measurement_cxl_only.sh" ]]; then
+            #    run_measurement_setup_baseline_default "${size}"
+            #    "${SCRIPT_DIR}/measurement_cxl_only.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "${workload_id}"
+            #else
+            #    echo "WARNING: ./measurement_cxl_only.sh not found or not executable; skipping CXL-only"
+            #fi
 
             run_measurement_teardown
         done
