--- editor:/users/zimooo2/.vscode-server/data/User/History/-198ec96b/MRqf.sh
+++ /users/zimooo2/.vscode-server/data/User/History/-198ec96b/xe5u.sh
@@ -4,10 +4,11 @@
 MEASUREMENT_WORKLOADS_PATH="${MEASUREMENT_WORKLOADS_DIR}/workloads"
 
 declare -ar MEASUREMENT_DEFAULT_WORKLOAD_IDS=(
-    "bc-kron.sg"
-    "pr-kron.sg"
-    "bc-twitter.sg"
-    "pr-twitter.sg"
+    
+    #"bc-kron.sg"
+    #"pr-kron.sg"
+    #"bc-twitter.sg"
+    #"pr-twitter.sg"
     #"XSBench"
     #
     #"faiss_10M-original"
