--- may04_git/measurement_workloads.sh
+++ may14_git/measurement_workloads.sh
@@ -4,14 +4,18 @@
 MEASUREMENT_WORKLOADS_PATH="${MEASUREMENT_WORKLOADS_DIR}/workloads"
 
 declare -ar MEASUREMENT_DEFAULT_WORKLOAD_IDS=(
-    "faiss_10M"
+    #"bc-kron.sg"
+    #"pr-kron.sg"
+    #"bc-twitter.sg"
+    #"pr-twitter.sg"
+    #"XSBench"
+    #
+    #"faiss_10M"
     #"mg.D.x"
-    #DuckDB-TPCH-sf100"
-    "bc-kron.sg"
-    "XSBench"
-    "pr-kron.sg"
-    "bc-twitter.sg"
-    "pr-twitter.sg"
+    #"DuckDB-TPCH-sf100"
+    "bc-twitter.sg-long"
+    "pr-twitter.sg-long"
+
     #"lulesh2.0_s400"
 
 
