--- 8d6a8bc66:measurement_workloads.sh
+++ -198ec96b/e8Wn.sh
@@ -4,14 +4,14 @@
 MEASUREMENT_WORKLOADS_PATH="${MEASUREMENT_WORKLOADS_DIR}/workloads"
 
 declare -ar MEASUREMENT_DEFAULT_WORKLOAD_IDS=(
-    "faiss_10M"
-    #"mg.D.x"
-    #DuckDB-TPCH-sf100"
     "bc-kron.sg"
-    "XSBench"
     "pr-kron.sg"
     "bc-twitter.sg"
     "pr-twitter.sg"
+    "faiss_10M"
+    "mg.D.x"
+    "DuckDB-TPCH-sf100"
+    "XSBench"
     #"lulesh2.0_s400"
 
 
