diff --git a/measurement_workloads.sh b/measurement_workloads.sh
index 5c6d506f9..ec1d98bc6 100755
--- a/measurement_workloads.sh
+++ b/measurement_workloads.sh
@@ -8,13 +8,13 @@ declare -ar MEASUREMENT_DEFAULT_WORKLOAD_IDS=(
     #"pr-kron.sg"
     #"bc-twitter.sg"
     #"pr-twitter.sg"
-    #"XSBench"
+    "XSBench"
     #
     #"faiss_10M"
     #"mg.D.x"
     #"DuckDB-TPCH-sf100"
-    "bc-twitter.sg-long"
-    "pr-twitter.sg-long"
+    #"bc-twitter.sg-long"
+    #"pr-twitter.sg-long"
 
     #"lulesh2.0_s400"
 
diff --git a/measurement_workloads_long.sh b/measurement_workloads_long.sh
index b1d5d6782..622087ddd 100755
--- a/measurement_workloads_long.sh
+++ b/measurement_workloads_long.sh
@@ -6,8 +6,8 @@ MEASUREMENT_WORKLOADS_PATH="${MEASUREMENT_WORKLOADS_DIR}/workloads"
 declare -ar MEASUREMENT_DEFAULT_WORKLOAD_IDS=(
     #"bc-kron.sg-long"
     #"pr-kron.sg-long"
-    "bc-twitter.sg-long"
-    #"pr-twitter.sg-long"
+    #"bc-twitter.sg-long"
+    "pr-twitter.sg-long"
     #"faiss_10M-long"
 
     #"XSBench"
diff --git a/run_all_measurements.sh b/run_all_measurements.sh
index c55096f01..b0ac884fb 100755
--- a/run_all_measurements.sh
+++ b/run_all_measurements.sh
@@ -56,8 +56,8 @@ EOF
     fi
 }
 
-SIZES=(6032)
-RUNS=3
+SIZES=(0)
+RUNS=10
 
 ARMS_LIB_SUFFIX=${ARMS_LIB_SUFFIX:-_plain}
 
@@ -66,7 +66,7 @@ mapfile -t WORKLOAD_IDS < <(measurement_list_default_workloads)
 for size in "${SIZES[@]}"; do
     echo "== Running measurements with size ${size}MiB =="
 
-    for run in $(seq 2 ${RUNS}); do
+    for run in $(seq 4 ${RUNS}); do
         echo "-- Run ${run}/${RUNS} for size ${size}MiB --"
 
         for workload_id in "${WORKLOAD_IDS[@]}"; do
@@ -80,8 +80,8 @@ for size in "${SIZES[@]}"; do
             #fi
 
             # ARMS
-            run_measurement_setup "${size}"
-            "${SCRIPT_DIR}/measurement_arms.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "" "${workload_id}"
+            #run_measurement_setup "${size}"
+            #"${SCRIPT_DIR}/measurement_arms.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "" "${workload_id}"
 
 
             # NOMAD
@@ -119,13 +119,13 @@ for size in "${SIZES[@]}"; do
             #    echo "WARNING: ./measurement_dram_only.sh not found or not executable; skipping DRAM-only"
             #fi
 
-            ## CXL-only baseline
-            #if [[ -x "${SCRIPT_DIR}/measurement_cxl_only.sh" ]]; then
-            #    run_measurement_setup_baseline_default "${size}"
-            #    "${SCRIPT_DIR}/measurement_cxl_only.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "${workload_id}"
-            #else
-            #    echo "WARNING: ./measurement_cxl_only.sh not found or not executable; skipping CXL-only"
-            #fi
+            # CXL-only baseline
+            if [[ -x "${SCRIPT_DIR}/measurement_cxl_only.sh" ]]; then
+                run_measurement_setup_baseline_default "${size}"
+                "${SCRIPT_DIR}/measurement_cxl_only.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "${workload_id}"
+            else
+                echo "WARNING: ./measurement_cxl_only.sh not found or not executable; skipping CXL-only"
+            fi
 
             run_measurement_teardown
         done
diff --git a/run_all_measurements_long.sh b/run_all_measurements_long.sh
index fc59b6bf3..ae3d7b857 100755
--- a/run_all_measurements_long.sh
+++ b/run_all_measurements_long.sh
@@ -56,7 +56,7 @@ EOF
     fi
 }
 
-SIZES=(6035)
+SIZES=(4035)
 RUNS=3
 
 ARMS_LIB_SUFFIX=${ARMS_LIB_SUFFIX:-_plain}
@@ -74,15 +74,15 @@ for size in "${SIZES[@]}"; do
 
 
             # Model
-            if [[ -x "${SCRIPT_DIR}/measurement_model.sh" ]]; then
-                "${SCRIPT_DIR}/measurement_model.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "" "${workload_id}"
-            else
-                echo "WARNING: ./measurement_model.sh not found or not executable; skipping model"
-            fi
+            #if [[ -x "${SCRIPT_DIR}/measurement_model.sh" ]]; then
+            #    "${SCRIPT_DIR}/measurement_model.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "" "${workload_id}"
+            #else
+            #    echo "WARNING: ./measurement_model.sh not found or not executable; skipping model"
+            #fi
 
             # ARMS
-            #run_measurement_setup "${size}"
-            #"${SCRIPT_DIR}/measurement_arms.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "" "${workload_id}"
+            run_measurement_setup "${size}"
+            "${SCRIPT_DIR}/measurement_arms.sh" "${PLATFORM_ARGS[@]}" "${size}" "${run}" "" "${workload_id}"
 
 
             # NOMAD
