--- 4da3d6c3f/model.h
+++ /users/zimooo2/.vscode-server/data/User/History/-15a4ddc8/GhLe.h
@@ -11,7 +11,4 @@
 struct page_info;
 
 double model_predict(struct data_row &row, struct page_info &page);
-void model_predict_batch(std::vector<struct data_row> &rows,
-                         const std::vector<std::shared_ptr<struct page_info>> &pages);
-void model_predict_batch_observe(std::vector<struct data_row> &rows,
-                                 const std::vector<std::shared_ptr<struct page_info>> &pages);+void model_predict_batch(std::vector<struct data_row> &rows, const std::vector<struct page_info *> &pages);