--- git:4da3d6c3f:model.cpp
+++ 0218ffa795731c64cafbd00e1520934dbe445e6a
@@ -75,6 +75,19 @@
     const double r_ratio_20 = round_to_6(build_ratio_feature(row.ewma_20_r, row.ewma_20));
     const double r_ratio_100 = round_to_6(build_ratio_feature(row.ewma_100_r, row.ewma_100));
 
+    //    "ewma_2",
+    //    "ewma_5",
+    //    "ewma_20",
+    //    "ewma_100",
+    //    "global_avg_accesses_model",
+    //    "group_neg_mean",
+    //    "group_pos_mean",
+    //    "group_0_mean",
+    //    "age",
+    //    "r_ratio_20",
+    //    "r_ratio_100",
+    //    "group_ewma5_var"
+
     features[i++] = round_to_6(row.ewma_2);
     features[i++] = round_to_6(row.ewma_5);
     features[i++] = round_to_6(row.ewma_20);
@@ -83,7 +96,7 @@
     features[i++] = group_neg_mean;
     features[i++] = group_pos_mean;
     features[i++] = group_0_mean;
-    features[i++] = static_cast<double>(row.age > 100 ? 100 : row.age);
+    features[i++] = row.age; // static_cast<double>(row.age > 100 ? 100 : row.age);
     features[i++] = r_ratio_20;
     features[i++] = r_ratio_100;
     features[i++] = round_to_6(row.group_ewma5_var);
