--- may04
+++ may14
@@ -100,23 +100,24 @@
      1ae:	8d 4a 78             	lea    0x78(%rdx),%ecx
      1b1:	c1 e9 03             	shr    $0x3,%ecx
      1b4:	f3 48 ab             	rep stos %rax,%es:(%rdi)
-     1b7:	31 c0                	xor    %eax,%eax
-     1b9:	48 c7 83 e8 02 00 00 	movq   $0x0,0x2e8(%rbx)
-     1c0:	00 00 00 00 
-     1c4:	66 89 83 18 01 00 00 	mov    %ax,0x118(%rbx)
-     1cb:	c7 83 14 01 00 00 00 	movl   $0x0,0x114(%rbx)
-     1d2:	00 00 00 
-     1d5:	c7 43 60 00 00 00 00 	movl   $0x0,0x60(%rbx)
-     1dc:	48 c7 83 00 03 00 00 	movq   $0x0,0x300(%rbx)
-     1e3:	00 00 00 00 
-     1e7:	c7 43 10 00 00 00 00 	movl   $0x0,0x10(%rbx)
-     1ee:	c6 83 1a 01 00 00 01 	movb   $0x1,0x11a(%rbx)
-     1f5:	48 c7 83 18 03 00 00 	movq   $0x0,0x318(%rbx)
-     1fc:	00 00 00 00 
-     200:	0f 11 83 20 01 00 00 	movups %xmm0,0x120(%rbx)
-     207:	0f 11 8b 08 03 00 00 	movups %xmm1,0x308(%rbx)
-     20e:	5b                   	pop    %rbx
-     20f:	c3                   	ret
+     1b7:	c7 43 60 00 00 00 00 	movl   $0x0,0x60(%rbx)
+     1be:	48 b8 00 00 00 00 00 	movabs $0x101000000000000,%rax
+     1c5:	00 01 01 
+     1c8:	48 c7 83 e8 02 00 00 	movq   $0x0,0x2e8(%rbx)
+     1cf:	00 00 00 00 
+     1d3:	48 c7 83 00 03 00 00 	movq   $0x0,0x300(%rbx)
+     1da:	00 00 00 00 
+     1de:	c7 43 10 00 00 00 00 	movl   $0x0,0x10(%rbx)
+     1e5:	48 89 83 14 01 00 00 	mov    %rax,0x114(%rbx)
+     1ec:	48 c7 83 18 03 00 00 	movq   $0x0,0x318(%rbx)
+     1f3:	00 00 00 00 
+     1f7:	0f 11 83 20 01 00 00 	movups %xmm0,0x120(%rbx)
+     1fe:	0f 11 8b 08 03 00 00 	movups %xmm1,0x308(%rbx)
+     205:	5b                   	pop    %rbx
+     206:	c3                   	ret
+     207:	90                   	nop
+     208:	0f 1f 84 00 00 00 00 	nopl   0x0(%rax,%rax,1)
+     20f:	00 
 
 0000000000000210 <page_info::max_model_score_history() const>:
      210:	0f b6 87 5c 01 00 00 	movzbl 0x15c(%rdi),%eax
@@ -321,8 +322,8 @@
      4aa:	f3 0f 10 4c 24 0c    	movss  0xc(%rsp),%xmm1
      4b0:	41 39 c4             	cmp    %eax,%r12d
      4b3:	41 0f 46 c4          	cmovbe %r12d,%eax
-     4b7:	f3 41 0f 10 94 86 10 	movss  0xe10(%r14,%rax,4),%xmm2
-     4be:	0e 00 00 
+     4b7:	f3 41 0f 10 94 86 60 	movss  0x960(%r14,%rax,4),%xmm2
+     4be:	09 00 00 
      4c1:	e8 00 00 00 00       	call   4c6 <page_info::push_model_score(float)+0x116>
 			4c2: R_X86_64_PLT32	adjusted_ewma(float, float, float)-0x4
      4c6:	f3 0f 11 85 60 01 00 	movss  %xmm0,0x160(%rbp)
@@ -1028,21 +1029,21 @@
      f78:	0f 1f 84 00 00 00 00 	nopl   0x0(%rax,%rax,1)
      f7f:	00 
 
-0000000000000f80 <page_info::update_can_promote(unsigned long, unsigned long)>:
-     f80:	48 39 f2             	cmp    %rsi,%rdx
-     f83:	73 3b                	jae    fc0 <page_info::update_can_promote(unsigned long, unsigned long)+0x40>
+0000000000000f80 <page_info::update_can_promote(unsigned long, unsigned long, unsigned long)>:
+     f80:	48 39 f1             	cmp    %rsi,%rcx
+     f83:	73 3b                	jae    fc0 <page_info::update_can_promote(unsigned long, unsigned long, unsigned long)+0x40>
      f85:	f3 0f 10 87 10 01 00 	movss  0x110(%rdi),%xmm0
      f8c:	00 
      f8d:	66 0f ef c9          	pxor   %xmm1,%xmm1
      f91:	0f 2f c1             	comiss %xmm1,%xmm0
-     f94:	76 3a                	jbe    fd0 <page_info::update_can_promote(unsigned long, unsigned long)+0x50>
+     f94:	76 3a                	jbe    fd0 <page_info::update_can_promote(unsigned long, unsigned long, unsigned long)+0x50>
      f96:	0f b7 87 18 01 00 00 	movzwl 0x118(%rdi),%eax
      f9d:	83 c0 01             	add    $0x1,%eax
      fa0:	66 89 87 18 01 00 00 	mov    %ax,0x118(%rdi)
      fa7:	66 83 f8 01          	cmp    $0x1,%ax
-     fab:	76 23                	jbe    fd0 <page_info::update_can_promote(unsigned long, unsigned long)+0x50>
+     fab:	76 23                	jbe    fd0 <page_info::update_can_promote(unsigned long, unsigned long, unsigned long)+0x50>
      fad:	0f 2f 87 14 01 00 00 	comiss 0x114(%rdi),%xmm0
-     fb4:	72 1a                	jb     fd0 <page_info::update_can_promote(unsigned long, unsigned long)+0x50>
+     fb4:	72 1a                	jb     fd0 <page_info::update_can_promote(unsigned long, unsigned long, unsigned long)+0x50>
      fb6:	c6 87 1a 01 00 00 01 	movb   $0x1,0x11a(%rdi)
      fbd:	c3                   	ret
      fbe:	66 90                	xchg   %ax,%ax
